src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -996,113 +996,17 @@ void SetUpZoneSizingArrays(EnergyPlusData &state)
     // from the specified OA method
     for (int CtrlZoneNum = 1; CtrlZoneNum <= state.dataGlobal->NumOfZones; ++CtrlZoneNum) {
         if (!state.dataZoneEquip->ZoneEquipConfig(CtrlZoneNum).IsControlled) continue;
-        auto const &thisZone = state.dataHeatBal->Zone(CtrlZoneNum);
         auto &finalZoneSizing = state.dataSize->FinalZoneSizing(CtrlZoneNum);
         auto &calcFinalZoneSizing = state.dataSize->CalcFinalZoneSizing(CtrlZoneNum);
-        // Use the max occupancy PEOPLE structure to calculate design min OA for each zone from the outside air flow per person input
-        Real64 TotPeopleInZone = 0.0;
-        Real64 ZoneMinOccupancy = 0.;
-        int DSOAPtr = finalZoneSizing.ZoneDesignSpecOAIndex; // index to DesignSpecification:OutdoorAir object
-        if ((DSOAPtr > 0) && !dsoaError) {
```

Comment:
Move a chunk of code to a separate function so it can be called for zones or spaces.

Reply:
:+1: